.github/copilot-instructions.md

```diff
@@ -0,0 +1,164 @@
+- **Content Presentation**: Cards, alerts, banners for content display
+- **Navigation**: Header, breadcrumbs, pagination components
+- **Data Display**: Tables, lists, and data visualization components
+
+## Common Development Tasks
+
+### Adding New Components
+1. Create component in `packages/web-components/src/components/`
+2. Add Stencil component with proper decorators and JSDoc
+3. Create comprehensive unit tests
+4. Add Storybook stories for documentation
+5. Generate React bindings automatically
+6. Update component documentation and design guidance
+
+### Updating Existing Components
+1. Maintain backward compatibility when possible
+2. Update unit tests and e2e tests
+3. Check for breaking changes and document them
+4. Update Storybook documentation
+5. Consider impact on consuming applications
+
+### Publishing Components
+1. Components are published via automated CI/CD
+2. Semantic versioning for release management
+3. Release notes generated from conventional commits
+4. Multiple npm packages published from monorepo
+
+## Integration and Usage
+
+### External Dependencies
+- **vets-website**: Primary consumer application
+- **content-build**: Static site generator for VA.gov
+- **design.va.gov**: Design system documentation site
+
+### Analytics and Metrics
+- **Dashboard CLI**: Tracks component usage across VA applications
+- **BigQuery Integration**: Usage data stored for analysis
+- **Weekly Reports**: Automated tracking of adoption and usage patterns
+
+## Development Best Practices
+
+### Code Quality
+- TypeScript for type safety
+- ESLint and Prettier for consistent formatting
+- Conventional commits for clear git history
+- Code review process for all changes
+
+### Documentation
+- Comprehensive JSDoc for all public APIs
+- Storybook stories with controls and examples
+- README files with usage instructions
+- Design guidance documentation
+
+### Performance
+- Lazy loading for web components
+- Optimized bundle sizes
+- Tree-shaking support
+- Minimal external dependencies
+
+## Troubleshooting Common Issues
+
+### Component Not Rendering
+- Check if component is properly imported
+- Verify Stencil build has completed
+- Check for JavaScript errors in console
+- Ensure proper HTML structure
+
+### Styling Issues
+- Check CSS custom properties are defined
+- Verify design token usage
+- Check for CSS specificity conflicts
+- Test in different browsers
+
+### Accessibility Problems
+- Run axe-core accessibility tests
+- Test with keyboard navigation
+- Verify screen reader compatibility
+- Check color contrast ratios
+
+## Resources and Links
+
+- **Design System Documentation**: [design.va.gov](https://design.va.gov)
+- **Component Library**: [design.va.gov/components](https://design.va.gov/components)
+- **GitHub Repository**: [department-of-veterans-affairs/component-library](https://github.com/department-of-veterans-affairs/component-library)
+- **Storybook Documentation**: Available in development builds
```

Comment:
> - **Storybook Documentation**: Available in development builds

Could we also link to Storybook?

https://design.va.gov/storybook/?path=/docs/about-introduction--docs